log-insights/setup/aiven/03_configure_collector.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Step 3: Configure the Collector'
+install_track_title: 'Log Insights: Setup (Aiven)'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+import imgGcpFirewallRules from './gcp-firewall-rules.png'
+import imgAwsSecurityGroup from './aws-security-group.png'
+
+export const ImgGcpFirewallRules = () => <img alt="GCP firewall rules allowing access on syslog port" src={imgGcpFirewallRules} />
+
+export const ImgAwsSecurityGroup = () => <img alt="AWS Security Group allowing access on syslog port" src={imgAwsSecurityGroup} />
+
+We can now configure the syslog receiver in the collector. Edit the collector
+config and add the
+[`db_log_syslog_server`](https://pganalyze.com/docs/collector/settings#self-managed-servers)
+setting. This should be set to a local address on any free port:
+
+```
+db_log_syslog_server: 0.0.0.0:32514
+```
+
+(If using Docker, use the `LOG_SYSLOG_SERVER` environment variable instead.)
+
+It's recommended to use an unprivileged port number (1024 and up) to avoid
+running the collector as root.
+
+Restart the collector.
+
+Make sure that the port you've selected is not blocked by a firewall or security group rules.
+E.g., in GCP, you'll need to add a firewall rule:
+
+<ImgGcpFirewallRules />
+
+In AWS, you'll need to add a security group rule:
+
+<ImgAwsSecurityGroup />
+
+You can verify that the port is accessible with telnet:
```

Comment:
Not a big deal, but on macOS I have `nc` but not `telnet` by default - maybe we should be using `nc` here?

```
lfittl@starfish pganalyze % nc -v 107.21.62.12 32514
Connection to 107.21.62.12 port 32514 [tcp/*] succeeded!
^C
lfittl@starfish pganalyze % nc -v 107.21.62.12 32515
^C
```

(if we keep telnet we may want to link https://superuser.com/questions/486496/how-do-i-exit-telnet -- as I had forgotten too how I would produce `^]`..)

Reply:
Ah, `nc` is a  great idea.